tests/everest/test_everserver.py

```diff
@@ -121,8 +117,9 @@ def test_everserver_status_failure(mf_1):
     "everest.detached.jobs.everserver.export_to_csv",
     side_effect=partial(check_status, status=ServerStatus.exporting_to_csv),
 )
-@tmpdir(relpath("..", "..", "test-data", "everest", "math_func"))
-def test_everserver_status_running_complete(*args):
+def test_everserver_status_running_complete(
+    _1, _2, _3, _4, _5, _6, _7, _8, _9, copy_math_func_test_data_to_tmp
```

Comment:
would this work?: `copy_math_func_test_data_to_tmp, *args`. Not sure if that is valid Python... Would it be desirable?

Reply:
I was not able to get any version with mixing fixtures and *args as parameters that's why I resorted to _*. I agree its ugly but I found no other way